app/src/main/java/org/jellyfin/androidtv/details/DetailsDescriptionPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package org.jellyfin.androidtv.details
+
+import androidx.leanback.widget.AbstractDetailsDescriptionPresenter
+
+class DetailsDescriptionPresenter(val title: String, val subtitle: String, val body: String) : AbstractDetailsDescriptionPresenter() {
+	override fun onBindDescription(viewHolder: ViewHolder, itemData: Any) {
+		val us = this
+
+		viewHolder.apply {
```

Comment:
When using `.also` the `this` scope won't change so you don't need the `val us = this` hack.